mappings/net/minecraft/world/gen/feature/EndCityFeature.mapping

```diff
@@ -1,2 +1,13 @@
 CLASS net/minecraft/unmapped/C_atkyhthn net/minecraft/world/gen/feature/EndCityFeature
+	FIELD f_sxrvdylj RANDOM_SALT I
```

Comment:
RANDOM_SALT is really funny to me for some reason. What does this mean though? I'm sure the name is accurate to what it does but I am not sure what salt means here.

Reply:
From wikipedia
> In cryptography, a salt is random data that is used as an additional input to a one-way function that hashes data, a password or passphrase

Reply:
makes sense then.